src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -622,6 +626,7 @@ protected function getKernelParameters(): array
      */
     protected function buildContainer(): ContainerBuilder
     {
+        // TODO: when removing getLogDir from KernelInterface, add check for LoggableKernelInterface
         foreach (['cache' => $this->getCacheDir(), 'build' => $this->warmupDir ?: $this->getBuildDir(), 'logs' => $this->getLogDir()] as $name => $dir) {
```

Comment:
i'd be curious if we could just remove initializing the log directory here.

monolog creates it lazy AFAIK for stream handlers.